orm/format.go

```diff
@@ -230,10 +230,7 @@ func (f Formatter) append(dst []byte, p *parser.Parser, params []interface{}) []
 			if !namedParamsOnce && len(params) > 0 {
 				namedParamsOnce = true
 				if len(params) > 0 {
```

Comment:
Looks like this check is not needed, because len is already checked on line 230. Can you remove it?